.gitignore

```diff
@@ -28,9 +29,12 @@
 *multisave
 *.archipelago
 *.apsave
-*.BIN
```

Comment:
This is bad. There are ROM files currently supported by Archipelago that are *.bin. I would recommend just explicitly inverting the ignore to include whatever file is needed.